config/express.js

```diff
@@ -47,7 +47,7 @@ if (config.env === 'development') {
 }
 
 // mount all routes on /api path
-app.use('/api', routes);
+app.use('/', routes);
```

Comment:
Ugh - I may have to take that back..headers may be the way to go if we want to maintain the HATEOAS style `"_links"` object